src/rapids_dependency_file_generator/schema.json

```diff
@@ -0,0 +1,160 @@
+ {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/rapidsai/dependency-file-generator/v1.0.0/src/rapids_dependency_file_generator/schema.json",
+    "type": "object",
+    "title": "RAPIDS Package Dependency Specification Format",
+    "description": "Consolidated specification of RAPIDS project dependencies.",
+    "properties": {
+        "files": {
+            "type": "object",
+            "patternProperties": {
+                ".*": {
+                    "type": "object",
+                    "properties": {
+                        "output": {"$ref": "#/$defs/Outputs"},
+                        "includes": {"type": "array", "items": {"type": "string"}},
+                        "matrix": {"$ref": "#/$defs/Matrix"},
+                        "requirements_dir": {"type": "string"},
+                        "conda_dir": {"type": "string"}
+                    },
+                    "additionalProperties": false,
```

Comment:
I think we want a minProperties of 1 here too

Reply:
Not needed since `output` and `includes` are actually required.

Reply:
Sorry I think I put this comment at the wrong indentation level. I meant for this to apply to the top-level `files` key. A `dependencies.yaml` file should always specify at least one file.

Reply:
Done.